code/script.js

```diff
@@ -1,14 +1,21 @@
-// Variables that point to selected DOM elements
-const chat = document.getElementById('chat');
+// (Pseudo code) Variables that point to selected DOM elements
+const chat = document.getElementById('chat')
+const inputWrapper = document.getElementById('input-wrapper')
+const input = document.getElementById('input') 
+const sendBtn = document.getElementById('send')
 
-// If you need any global variables that you can use across different functions, declare them here:
+let questionNumber = 1
 
+const botReply = (msg) => {
+  showMessage(msg, 'bot')
+}
 
-// Declare your functions after this comment
+const userReply = (msg) => {
+  showMessage(msg, 'user')
+}
```

Comment:
Interesting idea on wrapping 🎁  the show msg function, it makes it less error prone to write bot or user as a string wrong.